internal/repository/repository.go

```diff
@@ -0,0 +1,18 @@
+package repository
+
+import (
+	"github.com/newUser1337/task-news/internal/repository/article"
+	"github.com/newUser1337/task-news/internal/repository/mgdb"
+)
+
+type Repository struct {
+	Article *article.ArticleRepository
```

Comment:
is it a repo inside a repo? or `Repository` is a container for all other repositories?

Reply:
fixed